modules/sw360/sw360-workflow/src/test/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdaterTest.java

```diff
@@ -159,17 +187,15 @@ public void testGetLicensesWithNonExistingLicense() {
                 .setShortName(licenseName);
         License licenseAntenna = new License();
         licenseAntenna.setId(licenseName);
-        when(licenseClientAdapter.isLicenseOfArtifactAvailable(licenseName))
-                .thenReturn(false);
+        when(licenseClientAdapter.getLicenses())
+                .thenReturn(createTestLicenses());
         when(licenseClientAdapter.getSW360LicenseByAntennaLicense(licenseName))
                 .thenReturn(Optional.of(license));
         setUp(true, false);
 
         final Set<SW360License> licenses = metaDataUpdater.getLicenses(Collections.singletonList(licenseAntenna));
 
         assertThat(licenses).hasSize(0);
-
-        verify(licenseClientAdapter, times(1)).isLicenseOfArtifactAvailable(licenseName);
```

Comment:
Same: Assertion that getLicenses of LicenseClientAdapter is only called once

Reply:
Done